tests/gemini_test.yaml

```diff
@@ -0,0 +1,56 @@
+test_duration: 300
+n_db_nodes: 1
+n_loaders: 1
+n_monitor_nodes: 1
+user_prefix: 'gemini-test-3h-200gb'
+failure_post_behavior: keep
+ip_ssh_connections: 'private'
+experimental: 'true'
+instance_provision: 'spot_low_price'
+# instance_provision can be one of the following: on_demand|spot_fleet|spot_low_price|spot_duration
+store_results_in_elasticsearch: False
+
+# gemini
+# cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
+# the below cmd runs about 3 hours
+gemini_cmd: "gemini -d -n 1000000 -c 20 -p 100 -m mixed -f"
+gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
+gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+
+reuse_cluster: False
+db_nodes_public_ip: []
+db_nodes_private_ip: []
+oracle_db_nodes_public_ip: []
+oracle_db_nodes_private_ip: []
+loaders_public_ip: []
+loaders_private_ip: []
+monitor_nodes_public_ip: []
+monitor_nodes_private_ip: []
+
+backends: !mux
+    aws: !mux
+        cluster_backend: 'aws'
+        instance_type_loader: 'c4.4xlarge'
```

Comment:
Why do we need such a strong machine for loader?
Can't it be c4.large? or even c4.2xlarge?

Reply:
I think we can use c4.2xlarge